Scripts/build.sh

```diff
@@ -87,7 +87,7 @@ function xcbuild() {
 
 function fastlane_test() {
   if [[ -n "$XC_DESTINATION" ]]; then
-    SDK="$XC_SDK" DEST="$XC_DESTINATION" SCHEME="$1" bundle exec fastlane test
+    SDK="$XC_SDK" DEST="$XC_DESTINATION" SCHEME="$1" fastlane test
```

Comment:
shall we check first if fastlane is present using `which`? This might fail if we decide to chance CI once or run it locally

Reply:
Yes, good idea. Will add it.

Reply:
Added https://github.com/appium/WebDriverAgent/pull/559/files#diff-688f25f2178d0f34696efce56d6c90f8f89fa5e88ba61efbafa9a96a652b2ceaR89-R93.
After bundle install also work with only `fastlane` command.